recipes/prometheus/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set provider = "prometheus" %}
+{% set goname = "github.com/prometheus/"+provider.lower() %}
+{% set version = "2.12.0" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder:  {{ pkg_src }}
+    url: https://{{ goname }}/archive/v{{ version }}.tar.gz
+    sha256: 9bd9ae6df02777a9ba3f6f544338865861decabd02054aa64975449bd6009e5a
+
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - pushd {{ pkg_src }}/cmd/{{ name }}
+    - go build -v -o $GOBIN/{{ name }} .       # [unix]
+    - go build -v -o %GOBIN%\{{ name }}.exe .  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+
+test:
+  requires:
+    - m2-bash       # [win]
+    - m2-coreutils  # [win]
+
+  commands:
+    - prometheus --version
+
+about:
+  home: https://prometheus.io/
+  license: Apache-2.0
+  license_file: {{ pkg_src }}/LICENSE
+  summary: Prometheus, a Cloud Native Computing Foundation project, is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
```

Comment:
The `summary` should usually be short. You can use the `description` field to add additional details.

Reply:
Thanks, I have shortened the summary.